CONTRIBUTING.md

```diff
@@ -39,23 +39,47 @@ contribute is to learn how to use it and put your changes on a Git repository.
 There's a plenty of documentation about Git -- you can start with the [Pro Git
 book](http://git-scm.com/book/).
 
-### Forks + GitHub Pull requests
+
+### Setting up a Development Environment
+
+To set up your local development environment for contributing to the main
+repository [nltk/nltk](https://github.com/nltk/nltk/):
+
+- Fork the [nltk/nltk](https://github.com/nltk/nltk/) repository on GitHub
+  to your account;
+- Clone your forked repository locally
+  (`git clone https://github.com/<your-github-username>/nltk.git`);
+- Run `cd nltk` to get to the root directory of the `nltk` code base;
+- Install the dependencies (`pip install -r pip-req.txt`);
+- Install the `nltk` library in development mode
+  (`python setup.py develop`);
+- Download the datasets (`python -c "import nltk; nltk.download('all')"`) --
```

Comment:
Added this point about getting the NLTK datasets, because a few tests failed because of the lack of several datasets when I tried to run the tests. I put down `'all'` for convenience, but felt slightly uncomfortable because of wasting unnecessary resources (cf. #1787). If it'd be more desirable to download just those datasets one by one for the sake of running the tests (this would seem a little weird, but I don't have strong feelings), I'd be happy to single those out and update this PR.